### PR TITLE
Implement CREATE endpoint for OpeningBatch

### PR DESCRIPTION
**Pull request by linhkhanhhoang**

### CODE REVIEW

**Review by hoangr1010 — COMMENTED** [review]
LGTM!

**Review by hoangr1010 — APPROVED** [review]

File: server/controllers/OpeningBatchController.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    if (!deletedOpeningBatch) {
+      return res.status(404).json({ message: 'Opening Batch not found' });
+    }
+    res.status(200).json({ message: 'Opening Batch deleted successfully', data: deletedOpeningBatch });
+  } catch (error) {
+    res.status(400).json({ message: 'Error deleting Opening Batch', error: error.message });
+  }
+};
+
+module.exports = {
```
Comment (khangnh107) [inline server/controllers/OpeningBatchController.js:26]: Change to ES6 export for consistency with other files in the app

**Review by khangnh107 — CHANGES_REQUESTED** [review]
Make the following change for consistency with the rest of the app please!

File: server/controllers/OpeningBatchController.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    const deletedOpeningBatch = await OpeningBatch.findByIdAndDelete(id);
+    if (!deletedOpeningBatch) {
+      return res.status(404).json({ message: 'Opening Batch not found' });
+    }
+    res.status(200).json({ message: 'Opening Batch deleted successfully', data: deletedOpeningBatch });
+  } catch (error) {
+    res.status(400).json({ message: 'Error deleting Opening Batch', error: error.message });
+  }
+};
+
```
Comment (khangnh107) [inline server/controllers/OpeningBatchController.js:25]: Change to ES6 export for consistency with other files in the app

**Review by khangnh107 — CHANGES_REQUESTED** [review]
Make the following changes for consistency with the rest of the app please!

File: server/controllers/OpeningBatchController.js (hunk 1 above)
Comment (khangnh107) [inline server/controllers/OpeningBatchController.js:26]: Change to ES6 export for consistency please

**Review by khangnh107 — CHANGES_REQUESTED** [review]
Change this for consistency with other import/export in the app please!

File: server/controllers/OpeningBatchController.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+module.exports = {
+  createOpeningtBatch,
+  deleteOpeningBatch
+};
```
Comment (khangnh107) [inline server/controllers/OpeningBatchController.js:29]: Change the export format to ES6 format for consistency please!

**Review by khangnh107 — CHANGES_REQUESTED** [review]
Make the following change for consistency please!

**Pull request closed without merging**